magentoerpconnect/product_category.py

```diff
@@ -265,7 +250,7 @@ def parent_id(self, record):
         mag_cat_id = binder.to_openerp(record['parent_id'])
 
         if category_id is None:
-            raise MappingError("The product category with "
-                               "magento id %s is not imported." %
+            raise MappingError(_("The product category with "
+                                 "magento id %s is not imported.") %
```

Comment:
We have similar errors in community and odoo modules that contain translation _(...) for technical errors.